packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -85,11 +85,20 @@ contract DeployImplementationsInput is CommonBase {
         else revert("DeployImplementationsInput: unknown selector");
     }
 
+    function set(bytes4 sel, bytes32 _value) public {
+        if (sel == this.salt.selector) _salt = _value;
+        else revert("DeployImplementationsInput: unknown selector");
+    }
+
     function loadInputFile(string memory _infile) public pure {
         _infile;
         require(false, "DeployImplementationsInput: not implemented");
     }
 
+    function salt() public view returns (bytes32) {
+        return _salt;
```

Comment:
For example:
```suggestion
        require(_salt != bytes32(0), "DeployImplementationsInput: not set");
        return _salt;
```

Reply:
Same as https://github.com/ethereum-optimism/optimism/pull/11935#discussion_r1763182758